### PR TITLE
MCR-2629 allow web resource to access certain properties

### PR DESCRIPTION
**Pull request by sebhofmann**
[Link to jira](https://mycore.atlassian.net/browse/MCR-2629).

### CODE REVIEW

**Review by kkrebs — CHANGES_REQUESTED** [review]
please add javadoc and license header

**Review by kkrebs — DISMISSED** [review]

File: mycore-base/src/main/java/org/mycore/frontend/servlets/MCRConfigHelperServlet.java [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    
+    @Override
+    public void init() throws ServletException {
+        super.init();
+
+        updateJSON();
+
+        MCRConfiguration2.addPropertyChangeEventLister((changedProperty) -> {
+            String propertiesParameter = getPropertiesParameter();
+            for (String property : propertiesParameter.split(";")) {
```
Comment (yagee-de) [inline mycore-base/src/main/java/org/mycore/frontend/servlets/MCRConfigHelperServlet.java]: Separated by comma or by semicolon? It JavaDocs and code should match.
If you go for comma you can use org.mycore.common.config.MCRConfiguration2#splitValue

Comment (yagee-de) [reply mycore-base/src/main/java/org/mycore/frontend/servlets/MCRConfigHelperServlet.java]: As init parameter may not change you could prefixed properties and exact properties somewhere an reuse it. Helps to reduce redundant code.

File: mycore-base/src/main/java/org/mycore/frontend/servlets/MCRConfigHelperServlet.java [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+import java.util.Date;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+/**
+ * Servlet to map the properties defined in the init param "Properties" to a JSON object with the property name as key
+ * and the property value as value. The properties in the init param need to be comma separated and can end with * to
+ * match all properties which start with the string before *.
+ */
+public class MCRConfigHelperServlet extends MCRServlet {
```
Comment (yagee-de) [inline mycore-base/src/main/java/org/mycore/frontend/servlets/MCRConfigHelperServlet.java]: Extending MCRServlet seems not necessary. No user session, no database transaction. To much overhead. Try extending HttpServlet instead

**Review by yagee-de — CHANGES_REQUESTED** [review]

File: mycore-base/src/main/java/org/mycore/frontend/servlets/MCRConfigHelperServlet.java (hunk 1 above)
Comment (sebhofmann) [reply mycore-base/src/main/java/org/mycore/frontend/servlets/MCRConfigHelperServlet.java]: MCRConfiguration2#splitValue is meant for splitting property content, so its a different use case.

**Review by yagee-de — DISMISSED** [review]

**Pull request merged**